### PR TITLE
edits to make many things clearer

### PR DESCRIPTION
**Pull request by blairmacintyre**
Cleaned up a lot of the language to simplify it, and especially to make clear what anchors are and are not doing.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -1,75 +1,71 @@
...
+        1.  A real world object that the system is able to identify from the real world understanding. These elements have have a pose, but also include other information such as geometry. At the moment of the publication of this explainer ARKit is able to understand [ARPlaneAnchor](https://developer.apple.com/documentation/arkit/arplaneanchor), [ARFaceAnchor](https://developer.apple.com/documentation/arkit/arfaceanchor) and [ARImageAnchor](https://developer.apple.com/documentation/arkit/arimageanchor) as anchors.
 
+        While ARKit uses the concept of an anchor to represent the pose and the identified real world object, this explainer currently uses the term anchor to only represent an object with a poses that specifies it's location relative to the physical world. The [ARAnchor](https://developer.apple.com/documentation/arkit/aranchor) (1) base class in ARKit would be equivalent of the concept of an anchor in this explainer. Additional information about the representation of real world objects is out of the scope of this explainer. This differentiation between the concept of an anchor in ARKit and in the scope of this explainer is subtle but important.
```
Comment (judax) [inline explainer.md]: an object with a **poses**

File: explainer.md [hunk 2]
```diff
@@ -1,75 +1,71 @@
...
+        1.  An arbitrary pose in 3D space that needs to be updated relative to the physical world. Anchors that represent an arbitrary pose can be created and registered in ARKit. This is the same as the anchor concept in this explainer.
+        1.  A real world object that the system is able to identify from the real world understanding. These elements have have a pose, but also include other information such as geometry. At the moment of the publication of this explainer ARKit is able to understand [ARPlaneAnchor](https://developer.apple.com/documentation/arkit/arplaneanchor), [ARFaceAnchor](https://developer.apple.com/documentation/arkit/arfaceanchor) and [ARImageAnchor](https://developer.apple.com/documentation/arkit/arimageanchor) as anchors.
```
Comment (judax) [inline explainer.md]: **have have a pose** (it was even in the initial version)

File: explainer.md [hunk 3]
```diff
@@ -1,75 +1,71 @@
...
-
+Anchors could also represent entities with poses that:
 
 3.  Persist, meaning that the anchor is able to live between executions of the same application.
 4.  Are shared between different applications.
 
-This explainer focuses on just the concept of arbitrary anchors (1) but opening the possibility for real world understanding (2) mainly for API simplicity and to allow the developer to try to always do the right thing. Creating anchors when placing virtual objects once the system has some understanding of the real world around it (hitting on a real world object for example) is a very common and recommended practice.
-
-The reasons for this limited scope are:
+This explainer focuses on just the first concept of anchors (1) as specifying the pose of a location in the world, but leaves open the possibility for anchors knowing their relationship to semantically meaningful parts of the physical real world that the system has detected (2).
```
Comment (judax) [inline explainer.md]: Should we explicitly include that 3 and 4 are out of the scope of the explainer? I know it is done down below but maybe a quick reminder here would be in place?

File: explainer.md [hunk 4]
```diff
@@ -1,75 +1,71 @@
...
 
 3.  Persist, meaning that the anchor is able to live between executions of the same application.
 4.  Are shared between different applications.
 
-This explainer focuses on just the concept of arbitrary anchors (1) but opening the possibility for real world understanding (2) mainly for API simplicity and to allow the developer to try to always do the right thing. Creating anchors when placing virtual objects once the system has some understanding of the real world around it (hitting on a real world object for example) is a very common and recommended practice.
-
-The reasons for this limited scope are:
+This explainer focuses on just the first concept of anchors (1) as specifying the pose of a location in the world, but leaves open the possibility for anchors knowing their relationship to semantically meaningful parts of the physical real world that the system has detected (2). 
 
+Creating anchors in relation to the structure of the physical world around the user (such as if the system supports intersecting a ray with the the systems understanding of the physical world) is a very common practice practice that we want to support.
```
Comment (judax) [inline explainer.md]: with the **the** system**'s**

Comment (judax) [reply explainer.md]: practice **practice**

Comment (judax) [reply explainer.md]: nit: I would suggest not using the first person in the explainer. If I have done it in the original version, we should correct it.

File: explainer.md [hunk 5]
```diff
@@ -1,75 +1,71 @@
...
-*   Understanding real world objects are outside of the scope for this explainer. Currently, there is no specific proposal on how to expose real world understanding objects on the web. Whenever such proposal is agreed upon, the scope of the anchor API could evolve to support it. Even in that case, there will be still a need for arbitrary 3d pose anchors.
-*   Persisting and sharing anchors is outside of the current scope of this explainer as they add much complexity and because there are still not clear technical paths on how to resolve these scenarios in the current AR platforms.
-
-Anchors are intended to update poses as the system's real-world understanding improves - they are intended to improve precision. They **are not** intended to track moving objects.
+*   Arbitrary anchors are the most basic yet useful type of anchors. There is always a need, no matter the granularity of the real world understanding the system has, to create arbitrary 3D poses as anchors. Moreover, arbitrary 3d pose anchors can be used to represent positions relative to detect or tracked objects when world understanding is expended to include object detection and tracking.
```
Comment (judax) [inline explainer.md]: Moreover, **anchors that represent arbitrary 3d poses** can be used to represent positions relative to **detected** or tracked

Comment (judax) [reply explainer.md]: expended or expected?

File: explainer.md [hunk 6]
```diff
@@ -1,75 +1,71 @@
...
+*   Understanding the semantics of the physical world (such as detecting and tracking markers, images, faces or objects) is outside the scope of this explainer. Currently, there is no specific proposal on how to expose even basic understanding of the physical world to the web, much less more interesting objects. Whenever such proposal is agreed upon, the scope of the anchor API could evolve to support it. But even in that case, there will still be a need for anchors with arbitrary 3D poses.  Currently, anchors **are not** intended to track moving objects.
+*   Persisting and sharing anchors is outside of the current scope of this explainer, since platform-level anchors (used to implement the anchor concept) are opaque and not compatible across different platforms.
 
+Anchors are intended to maintain a pose that corresponds to a location in the physical world, and will be updated by the system as the it's understanding of the physical-world changes.
```
Comment (judax) [inline explainer.md]: as **the** it's understanding

File: explainer.md [hunk 7]
```diff
@@ -1,75 +1,71 @@
...
 
-Although most of the use cases of anchor creation might be related to real world understanding (placing the virtual race track on top of a real table for example), there are use cases to create anchors based on arbitrary poses that do not have any direct relationship with real world objects, but with the camera pose. Imagine placing a virtual HUD (Heads Up Display) in mid air in front of the user to access a quick menu in AR. The placement of the UI element could be at some specific comfortable distance in front of the user and an anchor would be needed to make sure the UI stays at the correct position/orientation even if the tracking changes underneath.
+Imagine a small race track with cars on it. The race track will be placed at a location world space, so an anchor should be created for it, but  the cars are positioned relative to the race track, and the whole race track should move together if the anchor is repositioned. A pose update for the race track's anchor will, in turn, update the cars on top of it.
```
Comment (judax) [inline explainer.md]: but  the cars (unnecessary space).

Comment (judax) [reply explainer.md]: I am not fond of this line:

 but  the cars are positioned relative to the race track, and the whole race track should move together 

I think I understand what you try to explain (that the race track and the cars should all move together, but I think this sentence does not explain that correctly. Let me know what you think.

File: explainer.md [hunk 8]
```diff
@@ -1,75 +1,71 @@
...
 
+Although most use cases for anchor creation might be related to real-world understanding (e.g., placing the virtual race track on top of a real table), there are also use cases where anchors are created based on an arbitrary pose relative to the user's head (i.e., the system camera), instead of a direct relationship to the physical world. Imagine placing a virtual HUD (Heads Up Display) in mid air in front of the user to access a quick menu in AR. The placement of the UI element could be at some specific comfortable distance in front of the user, and an anchor would be needed to make sure the UI stays at the correct position/orientation even if the world coordinates change while the menu is visible.
```
Comment (judax) [inline explainer.md]: an anchor would be needed to make sure -> an anchor should be used to make sure.

File: explainer.md [hunk 9]
```diff
@@ -1,75 +1,71 @@
...
 
+Two examples where Anchors might update as real-world understanding improves are:
 
-1.  The system gains a more precise understanding of where a physical object is, which affects the pose created relative to it. For example, if a pose was created relative to a plane perceived at 1m above sea level, and the system subsequently learns that the plane is actually 0.5m above sea level, then the Anchor can update the pose;
-1.  The system gains a more precise understanding of where camera location was at time of when anchor was created. For example, if a pose was created relative to camera position at time t=0.5 with position (1,1,1), and the system subsequently learns that the camera was actually at position (0,1,1) at time t=0.5, then the Anchor can update the pose accordingly.
+1.  The system gains a more precise understanding of where a physical object is, which affects the pose created relative to it. For example, if a pose was created relative to a plane perceived at 1m above the floor, and as the user moves around, the system refines this estimateto being 0.95m above the floor, then the Anchor pose will be updated;
```
Comment (judax) [inline explainer.md]: **estimateto** -> estimate to

File: explainer.md [hunk 10]
```diff
@@ -1,75 +1,71 @@
...
+1.  The system shift the world coordinate system used to specify the camera location. For example, if a pose was created (relative to the physical world, or to the camera) such that the world position is (1,1,1), and the system subsequently shifts it's world coordiantes such that the position that was formerly (1,1,1) is now (0.75,1,1), then the Anchor pose can updated accordingly.
```
Comment (judax) [inline explainer.md]: **shifts**

Comment (judax) [reply explainer.md]: change "can updated accordingly" to "must be updated accordingly"

**Review by judax — COMMENTED** [review]
Some corrections but overall LGTM.

File: explainer.md (hunk 3 above)
Comment (blairmacintyre) [reply explainer.md]: Sure.  I will add  "The two uses are currently out of the scope of the anchor proposal." right under the 3 and 4 item list.

File: explainer.md (hunk 4 above)
Comment (blairmacintyre) [reply explainer.md]: agreed.  Changed the end to "is expected to be a very common practice"

File: explainer.md (hunk 5 above)
Comment (blairmacintyre) [reply explainer.md]: detected.

File: explainer.md (hunk 7 above)
Comment (blairmacintyre) [reply explainer.md]: how about

"Imagine a small race track with cars on it. The race track will be placed at a location world space, so an anchor should be created for it, with  the cars positioned relative to the race track. A pose update for the race track's anchor will update the location of the race track and, in turn, the location of the cars position relative to it."

**Comment by blairmacintyre** [comment]
Ok, added fixes that you suggested (some of which are highlighted in the comment discussions above)

**Pull request merged**